keyboards/eve/meteor/rules.mk

```diff
@@ -0,0 +1,48 @@
+# Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# MCU name
+MCU = atmega32a
+PROTOCOL = VUSB
```

Comment:
Lines 18-25 can be removed. QMK sets all these rules based on the defined MCU as of #6253.

Reference: https://github.com/qmk/qmk_firmware/commit/405dea01bef215a0b79524e4ff74364f1bcfeadf#diff-0d8a65ac3a12cde32dc134d7a79a00cdR77